### PR TITLE
doc: update Recommended usage conventions for karmadactl, and fix a t…

### LINKED ISSUE #347: Fix a typo in `configure-controllers` document
Just found a typo in https://karmada.io/docs/administrator/configuration/configure-controllers#optinal-controllers.

optinal --> optional

### PR DESCRIPTION
**Pull request by yike21**
…ypo in configure-controllers document

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
If the documents are updated, users can more easily find the `karmadactl interpret --operation ReviseReplica` section.

**Which issue(s) this PR fixes**:
Fixes #347 
Fixes [#3401](https://github.com/karmada-io/karmada/issues/3401)

**Special notes for your reviewer**:
@Poor12

### CODE REVIEW

**Comment by yike21** [comment]
*Note that karmadactl needs to be based on the latest `kube library`.*

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md [hunk 1]
```diff
@@ -137,22 +137,9 @@ spec:
         end
     dependencyInterpretation:
       luaScript: >
+        local kube = require("kube")
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md:140]: For such enhanced changes, I think only the current version needs to be changed.

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md [hunk 2]
```diff
@@ -202,6 +189,12 @@ karmadactl interpret -f resourceinterpretercustomization.yaml --observed-file ob
 karmadactl interpret -f resourceinterpretercustomization.yaml --desired-file desired-deploy-nginx.yaml --observed-file observed-deploy-nginx.yaml --operation Retain
 ```
 
+#### 执行 ReviseReplica 规则
```
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md:192]: Do we need to apply this change in version-1.4?

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md (hunk 1 above)
Comment (yike21) [reply i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md:140]: Yes, it's more reasonable.

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md (hunk 2 above)
Comment (yike21) [reply i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md:192]: From one side, the documentation for karmada v1.4 is no longer actively maintained. 
From the other side, [V1.4.0](https://github.com/karmada-io/karmada/releases/tag/v1.4.0) was released on November 30, 2022, and [`pkg/karmadactl/interpret/interpret.go`](https://github.com/karmada-io/karmada/commits/master/pkg/karmadactl/interpret/interpret.go) had the full functionality of `karmadactl interpret` on November 28, 2022. Chronologically, the documentation can be updated.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**